src/components/Catalog.tsx

```diff
@@ -106,7 +106,7 @@ export const Catalog = ({ handleClose }: { handleClose: () => void }) => {
             <>
               <ToolbarItem style={{ padding: '0', marginRight: '0' }}>
                 <InputGroup>
-                  <SearchInput
+                  <SearchInput style={{ width: '250px' }}
```

Comment:
In general, this should be avoided, since it will generate a "new" change every time we define a new object on each render. Probably here is not a big deal, though.

Another way to approach this situation might be leveraging `flexbox` and letting the field fill the available space at all times.

Reply:
sure, thanks for the hint - will have a look how to do this better